### DISCUSSION TITLE
Is there a way to create a single -libs- folder for a set of reveal-js presentations?

### DISCUSSION THREAD

**Discussion by friosavila** [quarto-dev/quarto-cli#5487] [Q&A] [answered]
Is there a way to create a single -libs- folder for a set of reveal-js presentations?

**Comment by friosavila** [comment]
Im trying to understand if there is a way to create a single common lib folder that contains common files for a Reveal.js presentation.
Thank you

**Reply by cscheid** [reply]
(note that I've converted your report to a discussion question.) In the future, please use the issue navigator appropriately instead of filing bugs with no information. Thank you!

**Reply by friosavila** [reply]
Thank you!
Will more careful next time
So the problem is as follows.
Say I want to create 3 sets of presentations in the same folder.
ppt1 ppt2 and ppt3.
When I use revealjs to render them. 3 separate lib folders are created. 
To save space, I would like to know if there is any option to request a single "library" folder across all presentations.
Thank you

**Reply by cscheid** [reply, marked as answer]
If those presentations are all part of a single website, I believe that quarto already puts all of the files in a single location. If you're rendering each of them individually, then I believe the answer is no.
